scripts/wrapper/withdraw.js

```diff
@@ -107,7 +107,7 @@ module.exports = function (web3, artifacts) {
               return
             }
             const usdValue = await amountUSDValue(amount, tokenData, globalPriceStorage)
-
+            log(`USD value for ${amount} ${tokenData.symbol} on bracket ${bracketAddress} is ${usdValue}$`)
```

Comment:
It would be very nice if the scripts had different logging levels, so that a variable level of detail could be expressed in the logs.
However, since there is only one logging level I think this is too much information: it would just fill the terminal with unnecessary information when doing large withdrawals.